### DISCUSSION TITLE
Multiple camera output Screen layout in grid and store in different floder

### DISCUSSION THREAD

**Discussion by staRx06** [ultralytics/yolov5#10453] [Q&A]
Hello sir
I have trained one model for object detection and it run successfully with one camera IP address , but while giving multiple camera input ..the output screen is overlapping each others ..So  can you help me to arrange multiple screen in proper grid format and store the detected objects information in different folder

waiting for your positive answers
Thank you

**Comment by glenn-jocher** [comment]
@staRx06 you'd need to customize detect.py to suit your custom requirements.

**Reply by ghost** [reply]
Hi sir...
I also have same problem.... Can u please specify where should I customize it....or any reference for the same
